cogs/embeds.py

```diff
@@ -1,24 +1,142 @@
 import discord
 from discord.ext import commands
+import json
+import re
```

Comment:
As a stylistic nitpick (but also for consistency), put standard library imports at the top (followed by project dependency imports, then local module imports).